src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Scroll from 'react-scroll';
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
+  }
+
+  render() {
+    // const { submission, data } = this.props.form;
+    // const { response } = submission;
+    // const name = data.veteranFullName;
+
+    return (
+      <div>
```

Comment:
Please don't add wrapping divs, use `<>` and `</>` instead.

Reply:
done